pkgs/development/python-modules/celery-batches/default.nix

```diff
@@ -0,0 +1,34 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildPythonPackage,
+  setuptools,
+  celery,
+}:
+
+buildPythonPackage rec {
+  pname = "celery-batches";
+  version = "0.9";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "clokep";
+    repo = "celery-batches";
+    tag = "v${version}";
+    hash = "sha256-w7k8VPtJf9VRB8lJC/ENk3kIMPITd+qRIXm1KrCktgc=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [ celery ];
+
+  pythonImportsCheck = [ "celery_batches" ];
+
```

Comment:
Same here

Reply:
Testing this would require setting up a celery instance